File: ObjectProgrammingPython/Zwierzeta/owca.py
from Zwierzeta.zwierze import Zwierze


class Owca(Zwierze):
    def __init__(self, Swiat, x, y, czyZyje):
        super(Owca, self).__init__(Swiat, x, y, czyZyje)
        self.sila = 4
        self.inicjatywa = 4
        self.kolor = "pink"
        self.oznaczenie = "O"
        self.czyMozeRozmnazac = 0
        self.wiek = 0

    def Wypisz(self):
        return "Owca" + self.koncowka()

File: ObjectProgrammingJava/projekt po - 2/src/projekt/zwierzeta/Owca.java
package projekt.zwierzeta;

import projekt.Wypisanie;
import projekt.swiat.Swiat;

import java.awt.*;

public class Owca extends Zwierze implements Wypisanie {

    public Owca(Swiat swiat, int x, int y, boolean czyZyje){
        polozenieX = x;
        polozenieY = y;
        sila = 4;
        inicjatywa = 4;
        kolor = Color.pink;
        oznaczenie = 'O';
        this.swiat = swiat;
        this.czyZyje = czyZyje;
        this.czyMozeRozmnazac = 0;
        wiek = 0;
    }

    @Override
    public String Wypisz() {
        return "Owca" + getKoncowka();
    }
}
